### PR TITLE
webauthn enroll and login feature

### PR DESCRIPTION
**Pull request by karthikaselvaraj-okta**

### CODE REVIEW

File: samples/embedded-auth-with-sdk/src/test/java/env/Hooks.java [hunk 1]
```diff
@@ -148,6 +148,41 @@ public void assignMFAGroupBeforeScenario(Scenario scenario) {
...
+		groups.add("WebAuthn Required");
+
+		List<Group> groupList = client.listGroups()
+				.stream()
+				.filter(group -> groups.contains(group.getProfile().getName()))
+				.collect(Collectors.toList());
+		Assert.assertFalse(groupList.isEmpty());
+		groupList.forEach(group -> Page.getUser().addToGroup(group.getId()));
+	}
+	public void assignWebAuthnGroup() {
```
Comment (vijetmahabaleshwar-okta) [inline samples/embedded-auth-with-sdk/src/test/java/env/Hooks.java:164]: nit: add empty line after method ends and new method starts

File: samples/embedded-auth-with-sdk/src/test/java/env/Hooks.java [hunk 2]
```diff
@@ -148,6 +148,41 @@ public void assignMFAGroupBeforeScenario(Scenario scenario) {
...
+		if (Page.getA18NProfile() != null) {
+			logger.info("Searching for a user to be added to WebAuthn Required group: " + Page.getA18NProfile().getEmailAddress());
+			Optional<User> userToAdd = client.listUsers(Page.getA18NProfile().getEmailAddress(), null, null, null, null)
+					.stream().filter(x -> x.getProfile().getEmail().equals(Page.getA18NProfile().getEmailAddress())).findFirst();
+			if (userToAdd.isPresent()) {
+				List<String> groups = new ArrayList<>();
+				groups.add("WebAuthn Required");
```
Comment (vijetmahabaleshwar-okta) [inline samples/embedded-auth-with-sdk/src/test/java/env/Hooks.java:171]: Make "WebAuthn Required" string into a static variable like `WEBAUTHN_GROUP_NAME`. Easier to read and also this string is used in multiple places

**Review by vijetmahabaleshwar-okta — APPROVED** [review]

File: samples/embedded-auth-with-sdk/src/test/java/env/Hooks.java (hunk 1 above)
Comment (karthikaselvaraj-okta) [reply samples/embedded-auth-with-sdk/src/test/java/env/Hooks.java:166]: Added empty line

File: samples/embedded-auth-with-sdk/src/test/java/env/Hooks.java (hunk 2 above)
Comment (karthikaselvaraj-okta) [reply samples/embedded-auth-with-sdk/src/test/java/env/Hooks.java]: updated the suggested change

**Pull request merged**